To: vim_dev@googlegroups.com
Subject: Patch 7.3.200
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.200 (after 7.3.198)
Problem:    CTRL-D doesn't complete :lang.
Solution:   Add the missing part of the change. (Dominique Pelle)
Files:	    src/ex_docmd.c


*** ../mercurial/vim73/src/ex_docmd.c	2011-05-19 14:50:49.000000000 +0200
--- src/ex_docmd.c	2011-05-25 12:45:02.000000000 +0200
***************
*** 3861,3873 ****
  #if (defined(HAVE_LOCALE_H) || defined(X_LOCALE)) \
  	&& (defined(FEAT_GETTEXT) || defined(FEAT_MBYTE))
  	case CMD_language:
! 	    if (*skiptowhite(arg) == NUL)
  	    {
  		xp->xp_context = EXPAND_LANGUAGE;
  		xp->xp_pattern = arg;
  	    }
  	    else
! 		xp->xp_context = EXPAND_NOTHING;
  	    break;
  #endif
  #if defined(FEAT_PROFILE)
--- 3861,3884 ----
  #if (defined(HAVE_LOCALE_H) || defined(X_LOCALE)) \
  	&& (defined(FEAT_GETTEXT) || defined(FEAT_MBYTE))
  	case CMD_language:
! 	    p = skiptowhite(arg);
! 	    if (*p == NUL)
  	    {
  		xp->xp_context = EXPAND_LANGUAGE;
  		xp->xp_pattern = arg;
  	    }
  	    else
! 	    {
! 		if ( STRNCMP(arg, "messages", p - arg) == 0
! 		  || STRNCMP(arg, "ctype", p - arg) == 0
! 		  || STRNCMP(arg, "time", p - arg) == 0)
! 		{
! 		    xp->xp_context = EXPAND_LOCALES;
! 		    xp->xp_pattern = skipwhite(p);
! 		}
! 		else
! 		    xp->xp_context = EXPAND_NOTHING;
! 	    }
  	    break;
  #endif
  #if defined(FEAT_PROFILE)
*** ../vim-7.3.199/src/version.c	2011-05-25 12:09:46.000000000 +0200
--- src/version.c	2011-05-25 12:49:49.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     200,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
105. When someone asks you for your address, you tell them your URL.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
